web/templates/compose_control_buttons.hbs

```diff
@@ -8,7 +8,9 @@
     <a role="button" class="compose_control_button fa fa-video-camera video_link" aria-label="{{t 'Add video call' }}" tabindex=0 data-tippy-content="{{t 'Add video call' }}"></a>
     <a role="button" class="compose_control_button fa fa-smile-o emoji_map" aria-label="{{t 'Add emoji' }}" tabindex=0 data-tippy-content="{{t 'Add emoji' }}"></a>
     <a role="button" class="compose_control_button fa fa-clock-o time_pick" aria-label="{{t 'Add global time' }}" tabindex=0 data-tooltip-template-id="add-global-time-tooltip" data-tippy-maxWidth="none"></a>
-    <a role="button" class="compose_control_button compose_gif_icon {{#unless giphy_enabled }} hide {{/unless}} zulip-icon zulip-icon-gif" aria-label="{{t 'Add GIF' }}" tabindex=0 data-tippy-content="{{t 'Add GIF' }}"></a>
+    <div class="compose_control_button {{#unless giphy_enabled }} hide {{/unless}}" data-tippy-content="{{t 'Add GIF' }}">
+        <i class="compose_gif_icon zulip-icon zulip-icon-gif" role="button" aria-haspopup="true" tabindex="0" aria-label="{{t 'Add GIF' }}"></i>
```

Comment:
We probably want better names for these; I'm not sure whether `compose_control_button` / `compose_control_button_icon` or `compose_control_button_tooltip_wrapper` / `compose_control_button` is more appropriate.